docs/js/main.js

```diff
@@ -56,6 +55,22 @@ const showRecipes = function (recipes) {
     for (const recipe in recipes) {
         const recipeID = recipes[recipe].id;
         const title = recipes[recipe].name;
+        //show array of tags for each recipe
+        const allRecipeTags = recipes[recipe].tags;
+        //console.log(allRecipeTags);
+        //show each tag object
+        for (let tag in allRecipeTags) {
```

Comment:
This is a loop for objects, but allRecipeTags is an array.

Reply:
That makes sense, not the right loop

Reply:
You do need for .. in, I guess.  I think because it is an array of objects.